frontend/benefit/applicant/src/components/applications/forms/application/step1/ApplicationFormStep1.tsx

```diff
@@ -149,19 +150,33 @@ const ApplicationFormStep1: React.FC<DynamicFormStepComponentProps> = ({
           </StyledFormGroup>
         )}
         <StyledFormGroup>
-          <TextInput
-            id={fields.companyIban.name}
-            name={fields.companyIban.name}
-            label={fields.companyIban.label}
-            placeholder={fields.companyIban.placeholder}
-            onChange={formik.handleChange}
-            onBlur={formik.handleBlur}
+          <InputMask
+            mask={fields.companyIban.mask?.format ?? ''}
+            maskChar={null}
             value={formik.values.companyIban}
-            invalid={!!getErrorMessage(APPLICATION_FIELDS.COMPANY_IBAN)}
-            aria-invalid={!!getErrorMessage(APPLICATION_FIELDS.COMPANY_IBAN)}
-            errorText={getErrorMessage(APPLICATION_FIELDS.COMPANY_IBAN)}
-            required
-          />
+            onBlur={formik.handleBlur}
+            onChange={(e) => {
```

Comment:
My linter is complaining about missing type on these lines, interesting that it passes on this pr.

Reply:
Hmm, that's weird. It should be inferred. 
And it has the right type here: 
![image](https://user-images.githubusercontent.com/23040926/122220495-0b5f5200-ceb9-11eb-8f78-3cd383d6a216.png)

Reply:
Linter has some problems that are addressed mostly on my PR